app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -2,64 +2,58 @@ import React from 'react';
 import { connect } from 'react-redux';
 import axios from 'axios';
 import PropTypes from 'prop-types';
-import ActiveOrders from './ActiveOrders';
 import PastOrders from './PastOrders';
 import AddForm from './addForm/AddForm';
 import Tabs from '../tabs/Tabs';
 import Tab from '../tabs/Tab';
-import {
-  fetchInpatientCareSetting,
-  fetchOutpatientCareSetting,
-} from '../../actions/careSetting';
 import SearchDrug from '../searchDrug';
+import ActiveOrders from './ActiveOrders';
 
 export class SearchAndAddOrder extends React.Component {
   constructor(props) {
     super();
+
     this.state = {
-      careSetting: 'OutPatient',
+      careSetting: '',
     };
   }
 
-  componentDidMount() {
-    this.props.fetchInpatientCareSetting();
-    this.props.fetchOutpatientCareSetting();
-  }
-
-
   handleCareSettings = (careSetting) => {
     this.setState({ careSetting });
   }
   render() {
     return (
       <div className="body-wrapper">
         <Tabs careSetting={this.handleCareSettings}>
-          <Tab
-            tabName="Outpatient">
+          <Tab tabName="Outpatient">
             <SearchDrug />
+            <form className="sized-inputs css-form" noValidate>
+              <p className="input-position-class">
+                <label name="input-id">Input label</label>
+                <input id="input-id" />
+              </p>
+            </form>
+            <ActiveOrders
+              tabName="OutPatient"
+              careSetting={this.props.outpatientCareSetting}
+              location={this.props.location}
+            />
           </Tab>
-          <Tab tabName="Inpatient">
-            <SearchDrug />
+          <Tab tabName="InPatient">Tab 2 content
```

Comment:
I don't think we still need the statement `Tab 2 content`.

Reply:
Will remove this.
Not actually part of the current task.